app/src/main/res/values-fr/strings.xml

```diff
@@ -367,22 +367,22 @@
     <string name="node_nobookmark">%1$d meilleurs nœuds mis en favoris automatiquement</string>
     <string name="label_test">Tester</string><!--note: as in "Test a network connection"-->
 
-    <string name="send_address_resolve_bip70">Resolving Payment URI&#8230;</string>
-    <string name="send_address_not_bip70">Could not resolve Payment URI</string>
-    <string name="send_address_bip70">Resolved Payment URI &#x2714;</string>
+    <string name="send_address_resolve_bip70">Résolution de l\'URI de Paiement&#8230;</string>
```

Comment:
Not sure how this is used, but I would not put the P in Paiement in upper case. Same comment applies to lines below.

Reply:
I did put a upparcase P as the english version is using the upercase P. Not sure either.

Reply:
maybe @m2049r could tell us if this should be kept uppercase or not

Reply:
This depends by the language. English use a lot of upper case letters for stilistic reasons, but it's not like that in many other languages. Just do as you prefer, but keep it consistent :)

Reply:
As i did every string with upper case to keep it stilistic, i would keep it uppercase here too.  
But if @assylias wants, i could (later) pr a change to bring everything lowercase.

Reply:
You don’t typically use upper case in French except to start a sentence. But it’s not a big deal either. Your call!